src/app/utils/firebase.js

```diff
@@ -0,0 +1,152 @@
+import firebase from 'firebase';
+import { FIREBASE_CONFIG } from '../config';
+
+export const firebaseApp = firebase.initializeApp(FIREBASE_CONFIG);
+export const firebaseAuth = firebaseApp.auth();
+export const firebaseDb = firebaseApp.database();
+
+const FireBaseTools = {
+
+  /**
+   * Return an instance of a firebase auth provider based on the provider string.
+   *
+   * @param provider
+   * @returns {firebase.auth.AuthProvider}
+   */
+    getProvider: (provider) => {
+        switch (provider) {
+        case 'email':
+            return new firebase.auth.EmailAuthProvider();
+        case 'facebook':
+            return new firebase.auth.FacebookAuthProvider();
+        case 'github':
+            return new firebase.auth.GithubAuthProvider();
+        case 'google':
+            return new firebase.auth.GoogleAuthProvider();
+        case 'twitter':
+            return new firebase.auth.TwitterAuthProvider();
```

Comment:
We should remove Facebook, Github and Twitter provider code snippets from here too since we'll only want Google integration